source/_patterns/03-organisms/sections/share/share.js

```diff
@@ -0,0 +1,4 @@
+$("#print").on("click tap", function() {
```

Comment:
I'd love to see something more specific to "share" since it's entirely possible that some html element (other than in share) could have an ID of `#print`